
# HG changeset patch
# User Aleix Pol <aleixpol@kde.org>
# Date 1605614550 -3600
# Node ID fdf2c3c91e4b2f56ea37fcb9f9c8b2a734c8e12e
# Parent  7f15c4292e3eba520bf959cf2f45bf1c2cf0eb17
Include the necessary metadata for MyGNUHealth

The desktop file for launchers and the metainfo.xml file for software centers.

diff --git a/org.gnuhealth.my.desktop b/org.gnuhealth.my.desktop
new file mode 100644
--- /dev/null
+++ b/org.gnuhealth.my.desktop
@@ -0,0 +1,12 @@
+[Desktop Entry]
+Version=1.0
+Type=Application
+Name=MyGNUHealth
+GenericName=Personal Health Record
+Comment=GNU Health Personal Health Record
+Keywords=Healthcare;Medicine;Bioinformatics;Management;Enterprise;ERP;Framework;Client;
+Exec=mygnuhealth %u
+Icon=my-gnu-health
+Terminal=false
+MimeType=application/tryton;
+Categories=Office;Finance;
diff --git a/org.gnuhealth.my.metainfo.xml b/org.gnuhealth.my.metainfo.xml
new file mode 100644
--- /dev/null
+++ b/org.gnuhealth.my.metainfo.xml
@@ -0,0 +1,24 @@
+<?xml version="1.0" encoding="UTF-8"?>
+<component type="desktop-application">
+  <id>org.gnuhealth.my</id>
+
+  <name>MyGNUHealth</name>
+  <summary>GNU Health Personal Health Record</summary>
+
+  <metadata_license>CC0-1.0</metadata_license>
+  <project_license>LGPL-3.0-only</project_license>
+
+  <recommends>
+    <control>pointing</control>
+    <control>keyboard</control>
+    <control>touch</control>
+  </recommends>
+
+  <description>
+    <p>
+      Tracking personal health records.
+    </p>
+  </description>
+
+  <launchable type="desktop-id">org.gnuhealth.my.desktop</launchable>
+</component>
diff --git a/setup.py b/setup.py
--- a/setup.py
+++ b/setup.py
@@ -50,4 +50,9 @@
     packages=find_packages(),
     include_package_data=True,
     zip_safe=False,
+    data_files = [
+        ('share/applications', ['org.gnuhealth.my.desktop']),
+        ('share/metainfo', ['org.gnuhealth.my.metainfo.xml']),
+        ('share/icons/hicolor/512x512/apps/', ['mygnuhealth/images/my-gnu-health.png']),
+    ]
 )

